docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3202,23 +3202,57 @@ are explained in more depth in the table below.
 
 **OCR**
 
-Optical character recognition is when a character is misread for another character that 
-looks similar. A common examples is 'S' and '5'. In order to emulate 
-that, there is a `GeCo like corrupter and related list of possible changes in the ocr_variations_upper_lower csv found here <https://github.com/ihmeuw/vivarium_research_prl/tree/main/src/vivarium_research_prl/noise>`_. 
+Optical character recognition is when a string is misread for another string that 
+is visually similar. Some common examples are 'S' instead of '5' and 'm' instead of 'iii'.
 
-To implement this, select the strings eligible for noise and apply 
-the OCR noise function to all strings with the user defined token 
-error rate. 
+.. todo::
+  Both of these examples seem like they might be backward, though they come from the CSV below.
+  Are we interpreting the CSV backward? Should it be commutative?
+
+We have compiled `a list of possible OCR substitutions in a CSV file <https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/ocr-variations-upper-lower.csv>`,
+where the first column is the real string (which we call a "token") and the second column is what it could be misread as (a "corruption").
+The same token can be associated with multiple corruptions.
+
+.. todo::
+  Where does this list come from?
```

Comment:
I think it is copied from GeCO (and maybe reformatted), which is allowed, because you can't copyright data (right?)

Reply:
Is that true that you can't copyright data? How is GeCo licensed?

Reply:
https://law.stackexchange.com/questions/11359/can-you-copyright-data

Reply:
@zmbc Here's what @aflaxman wrote about his code:

> Informed by reading https://dmm.anu.edu.au/geco/flex-data-gen-manual.pdf but not looking at the sourcecode, since it might be in conflict with the license we end up using for this sim.

I see some of the .csv's referenced in the pdf, but I haven't tried finding them on the GeCo site for download.